diff --git a/NuGet.config b/NuGet.config
index ce3c339ea..543c25ab5 100644
--- a/NuGet.config
+++ b/NuGet.config
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear/>
+    <add key="dev" value="/dotnet/dist/packages" />
     <add key="dotnetfeed dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="dotnetfeed dotnet-coreclr" value="https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json" />
     <add key="myget.org dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
diff --git a/dependencies.props b/dependencies.props
index 008e59aab..13730f6fe 100644
--- a/dependencies.props
+++ b/dependencies.props
@@ -1,10 +1,10 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RyuJITVersion>5.0.0-alpha1.19453.1</RyuJITVersion>
-    <ObjectWriterVersion>1.0.0-alpha-28204-03</ObjectWriterVersion>
-    <CoreFxVersion>5.0.0-alpha1.19457.4</CoreFxVersion>
+    <RyuJITVersion>5.0.0-dev*</RyuJITVersion>
+    <ObjectWriterVersion>1.0.0-dev*</ObjectWriterVersion>
+    <CoreFxVersion>5.0.0-dev*</CoreFxVersion>
     <CoreFxUapVersion>4.7.0-preview6.19265.2</CoreFxUapVersion>
-    <MicrosoftNETCoreNativeVersion>5.0.0-alpha1.19453.1</MicrosoftNETCoreNativeVersion>
+    <MicrosoftNETCoreNativeVersion>5.0.0-dev*</MicrosoftNETCoreNativeVersion>
     <MicrosoftNETCoreAppPackageVersion>2.1.11</MicrosoftNETCoreAppPackageVersion>
     <MicrosoftDotNetTestSdkVersion>15.8.0</MicrosoftDotNetTestSdkVersion>
     <XUnitPackageVersion>2.4.1-pre.build.4059</XUnitPackageVersion>
